### PR TITLE
Test build on tf2.7

### PR DESCRIPTION
**Pull request by seanpmorgan**
# Description
Prepare for next release pinned to TF2.7

TODO:
- [ ] Cheery pick https://github.com/seanpmorgan/addons/pull/16 before the release

### CODE REVIEW

**Comment by bhack** [comment]
Green light here now

**Comment by bhack** [comment]
Upgrade to 2.7.0 final release

**Comment by bhack** [comment]
@googlebot I consent.

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+ARG IMAGE_TYPE=latest-cpu
```
Comment (fsx950223) [inline .devcontainer/Dockerfile:5]: This file should be removed.

Comment (fsx950223) [reply .devcontainer/Dockerfile:5]: https://github.com/keras-team/keras/blob/master/.devcontainer/Dockerfile This file doesn't have a MS Copyright.

Comment (bhack) [reply .devcontainer/Dockerfile:5]: As it was contributed by me in the Microsoft/Vscode repos let's check what is the @chuxel position on this for the CLA

Comment (bhack) [reply .devcontainer/Dockerfile:5]: P.s. the original PR was submitted in this repo before the vscode/MS one after the rejection:
https://github.com/tensorflow/addons/pull/1309/files

Comment (Chuxel) [reply .devcontainer/Dockerfile:5]: This is your Dockefile, so no need to include the copyright.

Comment (bhack) [reply .devcontainer/Dockerfile:1]: Ok I've remote the MS copyright here.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ARG USER_GID=$USER_UID
+
+# Options for common package install script - SHA updated on release
+ARG INSTALL_ZSH="false"
+ARG UPGRADE_PACKAGES="false"
+ARG COMMON_SCRIPT_SOURCE="https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/script-library/common-debian.sh"
+ARG COMMON_SCRIPT_SHA="dev-mode"
+
+# Configure apt and install packages
+RUN apt-get update \
```
Comment (bhack) [inline .devcontainer/Dockerfile:15]: @Chuxel Some section like this one was inspired by [your template](https://github.com/microsoft/vscode-dev-containers/blob/main/container-templates/dockerfile/.devcontainer/Dockerfile) that has no COPYRIGHT.
So I suppose it is fine and in the scope of the template to derive from that without using the MS copyright.

**Pull request merged**